examples/test_api.c

```diff
@@ -256,7 +255,7 @@ double getbw(char* name, char* buf, size_t size, int times)
 
       /* get the file name to write our checkpoint file to */
       char newname[SCR_MAX_FILENAME];
-      safe_snprintf(newname, sizeof(newname), "%s/%s", label, name);
+      snprintf(newname, sizeof(newname), "%s/%s", label, name);
```

Comment:
Please see https://github.com/LLNL/scr/pull/247#discussion_r512246635:

I would probably use snprintf rather than the home-grown scnprintf then use snprintf's return value to check if the buffer was actually too small and abort rather than continue with the truncated value. Ie:

```
int len = snprintf(dir, sizeof(dir), "%s/checkpoint_set_%d", prefix, id);
if((size_t)len >= sizeof(dir) scr_abort(blah);
```

otherwise this seems to only patch part of the issue.

Reply:
Thanks, @rhaas80 .  I'll add something like this in.

Reply:
Ok, those are now updated to match the above pattern.

It's a bit overkill since these example programs aren't intended to be production worthy.  There's a lot of error checking that is left out in places to reduce clutter/complexity in the example code.

Reply:
I realized that safe_snprintf was only used in the test programs, so now I moved this from scr_util.c to test_common.c and updated the examples to call it again.